Extensions/3D/JsExtension.js

```diff
@@ -902,6 +903,12 @@ module.exports = {
         .setLabel(_('Depth'))
         .setMeasurementUnit(gd.MeasurementUnit.getPixel())
         .setGroup(_('Default size'));
+      objectProperties
+        .getOrCreate('cubeColor')
```

Comment:
I think you can name it simply `color`, we can deduce from the context that it's for the cube